File: app/backend/RememoryApp/Rememory.Persistance/Entities/Journey.cs
using MongoDB.Bson.Serialization.Attributes;

namespace Rememory.Persistance.Entities;

public class Journey : IDatabaseEntity
{
    [BsonId]
    public Guid Id { get; set; }
    public Guid UserId { get; set; }
    public string? Title { get; set; }
    public DateTime Start { get; set; }
    public DateTime End { get; set; }
}

File: app/frontend/src/models/journey.ts
import dayjs from 'dayjs';

export class Journey {
    public id: string;
    public userId: string;
    public title: string;
    public start: dayjs.Dayjs;
    public end: dayjs.Dayjs;
  
    public constructor (id: string, userId: string, title: string, start: dayjs.Dayjs, end: dayjs.Dayjs) {
        this.id = id;
        this.userId = userId;
        this.title = title;
        this.start = start;
        this.end = end;
    }
}
  